glslc/README.asciidoc

```diff
@@ -12,6 +12,7 @@
 ----
 glslc [-c|-S|-E]
       [-x glsl] [-std=standard]
+      [-x hlsl]
```

Comment:
I think we should write something like `[-x ...]` here to be consistent with the following.

BTW, `-std` should mention that it is not applicable to `-x hlsl` (if this is correct).